src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: "Avenir Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  font-weight: 300;
```

Comment:
keep in * selector only for seting box-sizing property. do not set other properties with this selector to avoid lack of perfomance